### PR TITLE
Update istio/istio test reference

### PR DESCRIPTION
**Pull request by ericvn**
This change is required because of https://github.com/istio/istio/pull/27793

[ ] Configuration Infrastructure
[ ] Docs
[ ] Installation
[ ] Networking
[ ] Performance and Scalability
[ ] Policies and Telemetry
[ ] Security
[ ] Test and Release
[ ] User Experience
[ ] Developer Infrastructure

### CODE REVIEW

**Comment by ericvn** [comment]
The test does pass locally.

File: content/en/docs/ops/diagnostic-tools/istioctl-analyze/index.md [hunk 1]
```diff
@@ -22,10 +22,10 @@ $ istioctl analyze --all-namespaces
 
 And that’s it! It’ll give you any recommendations that apply.
 
-For example, if you forgot to enable Istio injection (a very common issue), you would get the following warning:
+For example, if you forgot to enable Istio injection (a very common issue), you would get the following information message:
```
Comment (shamsher31) [inline content/en/docs/ops/diagnostic-tools/istioctl-analyze/index.md:25]: nit: How about just say `Info` message? This way user will be able to match with a command line output.
```suggestion
For example, if you forgot to enable Istio injection (a very common issue), you would get the following `Info` message:
```

**Review by shamsher31 — APPROVED** [review]
@ericvn Thanks

**Review by frankbu — APPROVED** [review]

**Pull request merged**